easybuild/easyconfigs/q/Qt/Qt-4.8.6-intel-2015b-GLib-2.44.1.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Qt'
+version = '4.8.6'
+
+homepage = 'http://qt-project.org/'
```

Comment:
recent Qt easyconfigs use `qt.io`

some other (style) changes are also missing in this easyconfig